README.md

```diff
@@ -1,5 +1,20 @@
-## vcpkg Website Sources
+# vcpkg website
 
-To regenerate the website, run `scripts/rebuild.sh` from within the docker container defined by `.devcontainer/Dockerfile`.
+## Requirements:
+- Docker
+- Node, npm
 
+## Build the website
+To regenerate the website, run `scripts/rebuild.sh` from within the docker container defined by `.devcontainer/Dockerfile`:
+```
+docker build .devcontainer -t "vcpkg-io"
+docker run -it --mount type=bind,source=<working directory>,target=/dst "vcpkg-io" bash dst/scripts/rebuild.sh
+```
 The commit of https://github.com/Microsoft/vcpkg used is defined within the `rebuild.sh` script.
```

Comment:
```suggestion
The `rebuild.sh` script can be modified to build a specific commit in this repository.
```

I know it's not part of the changes in this PR, but I feel like this line needs clarification.